app/src/androidTest/java/org/systers/mentorship/LoginActivityTest.kt

```diff
@@ -0,0 +1,160 @@
+package org.systers.mentorship
+
+import android.support.annotation.IdRes
+import android.support.design.widget.TextInputLayout
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.ViewInteraction
+import android.support.test.espresso.action.ViewActions.*
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.matcher.ViewMatchers.*
+import android.support.test.rule.ActivityTestRule
+import android.support.test.runner.AndroidJUnit4
+import android.view.View
+import android.widget.EditText
+import org.hamcrest.CoreMatchers.allOf
+import org.hamcrest.CoreMatchers.not
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.systers.mentorship.view.activities.LoginActivity
+
+/**
+ * This class specifies the UI test for LoginActivity
+ */
+@RunWith(AndroidJUnit4::class)
+class LoginActivityTest {
+    private val EMPTY_USERNAME_ERROR: String = "Username cannot be empty"
+    private val EMPTY_PASSWORD_ERROR: String = "Password cannot be empty"
+
+    private val TEST_USERNAME: String = "testUser"
```

Comment:
```suggestion
    private val TEST_USERNAME: String = "test User"
```

It's best if you put a space in between, because I'm sure username's with spaces are not allowed. so that in the future if anyone creates a user with username: "testUser" this test does not fail